express-postgres/server.ts

```diff
@@ -0,0 +1,82 @@
+import pg from 'pg';
+import express from 'express';
+import { ClientError, errorMiddleware } from './lib/index.js';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/pagila',
+  ssl: {
+    // Allow non-SSL traffic to localhost
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+
+app.get('/api/films', async (req, res, next) => {
+  try {
+    const sql = `
+    select "title",
+           "fimId",
+    from "films"
+    order by "replacementCost desc;
```

Comment:
This has a syntax error (missing `"`). Did you run this endpoint?

Reply:
i did fix those, i forgot to push the changes, will do that now.